pandas/tests/groupby/test_timegrouper.py

```diff
@@ -273,6 +273,21 @@ def test_timegrouper_with_reg_groups(self):
                 'whole_cost'].sum()
             assert_series_equal(result2, expected)
 
+    def test_groupby_resample_equivalence(self):
+        # Currently failing
```

Comment:
this is failing?

Reply:
It's a simplification of `test_timegrouper_with_reg_groups`, they're failing for the same reason (https://github.com/pandas-dev/pandas/pull/18921#issuecomment-353746002)